frontend/src/components/DownloadAnalysisLink/DownloadAnalysisLink.tsx

```diff
@@ -123,11 +125,25 @@ function getHeadersForCensusCategories(method: Method, censusCategories: string[
       ))))
 }
 
-function getDataForCensusCategories(serviceAreas: string[] | null, censusCategories: string[], store: Store) {
+/**
```

Comment:
no commented out code ever please

Reply:
I think it's knowledge worth keeping. How do you suggest keeping this information?

Reply:
What's wrong with leaving the function un-commented but also un-called?